Controls/Controls.js

```diff
@@ -771,6 +771,27 @@ define([
             control._title = settings.title || '';
 
 
+            Object.defineProperties(control, {
+                defaultOption: { writable: true },
+                _defaultOptionEl: {
+                    get: function () {
+                        if (!control.defaultOption) {
+                            return "";
+                        }
+
+                        var optionEl = document.createElement("option");
+                        optionEl.value = "";
+                        optionEl.disabled = true;
```

Comment:
Why is disabled true? If the DropList is used as a filter, the default option is used to disable the filter and setting disabled to true breaks this feature.

Reply:
Example: in the DateRange below